src/com/seafile/seadroid2/cameraupload/CameraUploadService.java

```diff
@@ -267,6 +267,18 @@ protected File doInBackground(Void... params) {
 
         @Override
         protected void onPostExecute(File photo) {
```

Comment:
The user could also change from wifi to mobile network while the task is active.